docs/docs/tools.mdx

```diff
@@ -10,6 +10,59 @@ keywords:
   - state
 ---
 
-Coming soon. 
+##### Table of Contents
 
-See the [tldraw repository](https://github.com/tldraw/tldraw/tree/main/apps/examples) for an example of how to create custom tools in tldraw.
+- [Introduction](#introduction)
+- [Types of tool](#types-of-tool)
+- [Inside a tool](#inside-a-tool)
+- [Handling events](#handling-events)
+
+## Introduction
+
+In tldraw, a **tool** is what we call any top-level state in our state chart. For example, the select tool, the draw tool, and the arrow tool are all top-level states that the user can be in.
```

Comment:
Let's establish that there's nothing 'special' about tools. It's just what we use to refer our the 'first-level' state nodes.